src/partials/reviews.html

```diff
@@ -0,0 +1,49 @@
+<section id="reviews" class="section reviews">
+    <div class="container">
+    <h2 class="reviews-title">Reviews</h2>
+    <div class="reviews-container">
+        <ul class="reviews-list">
+        <li class="reviews-item">
+            <img 
+                srcset="/img/customer/manone.webp 1x, /img/customer/mantwo.webp 2x"
+                alt="man with dark hair" class="reviews-image"
+                width="56"
+                height="56"/>
+            
+    <h3 class="reviews-subtitle">John Wax</h3>
+    <p class="reviews-text">Buying a watch from was the best choice for me. 
+        The design is perfectly combined with high quality materials. 
+        The strap fits nicely on the hand, and the mechanism works flawlessly.
+    </p>
+    </li>
+    <li class="reviews-item reviews-item-victoria">
+        <img 
+                srcset="/img/customer/girlone.webp 1x, /img/customer/girltwo.webp 2x"
```

Comment:
we should add one dot to the way because we have an extra folder for photos
for all photos - "./img